modules/light-clients/06-solomachine/client_state_test.go

```diff
@@ -802,11 +802,6 @@ func (suite *SoloMachineTestSuite) TestGetTimestampAtHeight() {
 			expValue:    suite.solomachine.ClientState().ConsensusState.Timestamp,
 			expPass:     true,
 		},
-		{
-			name:        "get timestamp at height not exists",
-			clientState: suite.solomachine.ClientState(),
-			height:      suite.solomachine.ClientState().GetLatestHeight().Increment(),
-		},
```

Comment:
this test case was testing the behaviour of `GetTimestampAtHeight` which would error out when `!cs.GetLatestHeight().EQ(height)` , which means a zero proof would never be valid. 

I'm not sure exactly how to go ahead with this test case or if it's still valid. cc @damiannolan @colin-axner